### ISSUE TITLE
Organisation Page: Small size and low contrast for social media icons on information page

### ISSUE THREAD

**Issue by rittermo** [liqd/adhocracy-plus#2927]
**URL:** https://adhocracy.plus/liqd/information/
**user:** participant
**expected behaviour:** The social media icons on the page should be easy to see and click
**behaviour:** The icons’ colors make them hard to see on grey background and they are small to click, especially using fingers on touch devices
**important screensize:** all screen sizes
**device & browser:** MacBook Pro 14, Safari 18.1.1.
**Comment/Question:** 

Screenshot?
<img width="1388" alt="Bildschirmfoto 2025-01-16 um 10 53 22" src="https://github.com/user-attachments/assets/a617fd0e-9e18-41a7-a3ff-df48a46f5952" />

**Comment by t-j-q** [comment]
Thanks!

**Comment by arc64** [comment]
@t-j-q Are you able to tell me how the information page is created? is it in wagtail cms? I've made a code fix for this but haven't tested it yet.

**Comment by t-j-q** [comment]
@arc64 You can find it in the [initiator backend](https://adhocracy.plus/liqd/dashboard/organisations/projects/). Select organisation settings (top right).
